src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,137 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+
+    private int capacity;
+    private int size;
+    private Object[] elementDate;
+
+    public ArrayList() {
+        this.elementDate = new Object[DEFAULT_CAPACITY];
+        capacity = DEFAULT_CAPACITY;
+    }
+
+    private Object[] grow() {
```

Comment:
`grow()` and `grow(int)` methods have similar code. 

you can reuse method `grow(int)` inside `grow()`